front-end/src/model/Task.ts

```diff
@@ -1,9 +1,11 @@
 export class Task {
```

Comment:
itt 2 probléma is van:
1. a deklarációknál csak a típus legyen megnevezve, értékadás ne töörténjen
1. az értékadás a konstuktorban kell megtörténjen